module/notifications/push/helper.js

```diff
@@ -8,31 +8,65 @@
 
 // dependencies
 
-const fcmNotification = require('fcm-notification');
+var admin = require("firebase-admin");
```

Comment:
@priyanka-TL - Make it const.

Remove other FCM related files to avoid confusion.

Remove fcm-notification package if not required.